framework/src/testing/block_processing_env.ts

```diff
@@ -256,6 +256,7 @@ export const getBlockProcessingEnv = async (
 		genesisBlock.header.id,
 		appConfig.genesisConfig.communityIdentifier,
 	);
+	removeDB(params.options?.databasePath);
```

Comment:
🤔 the cleanup is doing removeDB, not sure why it doesn't work

Reply:
in particular instance, it was not removed properly and never able to rerun the test anymore, so i fixed it by ensuring the DB don't exist